src/App.tsx

```diff
@@ -1,35 +1,181 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import React, { useState, ChangeEvent, useEffect } from 'react';
+import Header from './components/Header';
+import SearchBar from './components/SearchBar';
+import PendingRequests from './components/PendingRequests';
+import MyConnections from './components/MyConnections';
+import { ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import {
+  sendConnectionRequest,
+  getPendingConnectionRequests,
+  findConnectionsByUserId,
+  acceptConnectionRequest,
+  rejectConnectionRequest,
+  removeConnection,
+  searchUser,
+} from './services/api/connections.service';
+import './styles/App.css';
+import './styles/ToastStyles.css';
```

Comment:
I think we are using bootstrap to handle all the styles, unless there are custom styles that we need

Reply:
to prioritize Bootstrap for consistency. I’ll review the existing custom styles and see if they can be adjusted or replaced with Bootstrap classes. If certain styles are truly unique and not covered by Bootstrap, I’ll make sure they are well-documented and applied consistently across the project